src/server.js

```diff
@@ -452,6 +555,47 @@ api.post('/getPathNames', (req, res) => {
   });
 });
 
+api.post('/getBedRegions', (req, res) => {
+  console.log('received request for bedRegions');
+  const result = {
+    bedRegions: []
+  };
+
+  let bed_info = {chr:[], start:[], end:[], desc:[], chunk:[]};
+
+  if(req.body.bedFile != 'none'){
+    const bedFile =
+	  req.body.isUploadedFile === 'true'
+	  ? `./${req.body.bedFile}`
+	  : `${MOUNTED_DATA_PATH}${req.body.bedFile}`;
```

Comment:
We're just letting everybody on the Internet read every TSV on the system here, such as for example `/etc/shadow`; we need to make sure that this file is actually in the directory it is supposed to be in when we read it.

Reply:
It couldn't really read those kind of files because of permission issues (I tried). Anyway, [now](https://github.com/vgteam/sequenceTubeMap/pull/140/commits/8db286b9f20863a16979c82155aa6532bd04a9ae) there are checks to make sure the files have the proper extensions (in this case `.bed`). And also the path can have `..` or `/etc`, `/sys`, `/usr` in them, just in case.